resteasy-core/src/main/java/org/jboss/resteasy/core/ContextServletOutputStream.java

```diff
@@ -0,0 +1,175 @@
+package org.jboss.resteasy.core;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.WriteListener;
+
+public class ContextServletOutputStream extends ServletOutputStream
```

Comment:
This should be package private.

Reply:
Ok

Reply:
We should just make the class itself package private too IMO. No need to expose it :)